app/controllers/subscriber_lists_controller.rb

```diff
@@ -25,7 +25,7 @@ def show
 
   def create
     subscriber_list = CreateSubscriberListService.call(params: params, user: current_user)
-    render json: subscriber_list.to_json, status: :created
+    render json: subscriber_list.to_json
```

Comment:
I thought about whether this should return different codes depending on create vs update vs "found". I think you're right that it's not worth differentiating at this point.